bundle/FieldHandler/EnhancedBinaryFileHandler.php

```diff
@@ -28,23 +30,23 @@ public function __construct(IOServiceInterface $IOService)
     }
 
     /**
-     * @inheritDoc
+     * {@inheritdoc}
      */
     public function supports(Value $value)
     {
         return $value instanceof EnhancedBinaryFileValue;
     }
 
     /**
-     * @inheritDoc
```

Comment:
Can you do a PR without fixes to CS, to make more easier to review? We'll fix CS, after merging it.

Reply:
@emodric ok, done